src/api/types/metadata.d.ts

```diff
@@ -1023,10 +1023,6 @@ export interface operations {
             query?: {
                 assay?: string | null;
                 coverage?: number | null;
-                /** @description Filter based on 'coverage' that is greater than or equal to the given value. */
```

Comment:
Perhaps this is outdated with my recent commit, these fields should exist in the current API.

Reply:
I just generate again with latest orcabus code, it seems no changes, could you have a check again?